pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -301,6 +311,40 @@ private bool ValidateImportResultsArgs()
             return true;
         }
 
+        public static readonly Argument ARG_LIMIT_NOISE = new DocArgument(@"limit-noise", NUM_VALUE,
```

Comment:
Too general a name. Should be --chromatograms-limit-noise or --chrom-limit-noise